src/lib/string.ts

```diff
@@ -21,3 +21,13 @@ export function slugify(str: string) {
 
     return str.slice(0, 50)
 }
+
+export function randomString(length: number) {
```

Comment:
We're using this to mock fake data, but we're already pulling in faker so we can prob use that in place of this, probably `faker.string.alpha(8) // 'fEcAaCVbaR'`

Reply:
hm, maybe but I like this because it skips easily confused letters like `0O` etc

Reply:
there is an [exclude characters](https://fakerjs.dev/api/string#alpha) in the options for this too if thats a concern